amida-tech/amida-survey-microservice
# Feature/replicate ser 18 for assessment answers

#### What's this PR do?
adds a parameter to the '/assessment-answers' endpoint to replicate the parameter added in [ser-18](https://jira.amida-tech.com/browse/SER-18) to  filter the returned assessment answers list by user-progress ('new', 'in-progress', or 'completed')

#### Related JIRA tickets:

[SER-94](https://jira.amida-tech.com/browse/SER-94)

#### How should this be manually tested?

try to make requests with the param and check if it returns only those surveys whose status matches the param value. Put in junk values for the param to make sure it doesn't cause a 500 error. 

#### Any background context you want to provide?
#### Screenshots (if appropriate):
